about.html

```diff
@@ -28,120 +28,20 @@ <h1>Sandstorm is on a mission</h1>
 
-<section id="advisors">
-<h2>Advisors</h2>
-  <ul class="advisors">
-    <li class="swetland">
-      <h3>Brian Swetland</h3>
-      <h4>Operating Systems Expert</h4>
-      <p>As Android's Systems / Kernel Lead from 2004 to 2012, Brian built the lower levels of what is now the world's most-popular operating system. Prior to that, Brian worked as a kernel engineer for BeOS and Danger, Inc.
-    <li class="lutomirski">
-      <h3>Andrew Lutomirski</h3>
-      <h4>Kernel Security Expert</h4>
-      <p>Andy is a Linux kernel developer focused on security. Every month or two, he sends us a message: "I found another Linux kernel vulnerability, but Sandstorm's seccomp filter blocks it." That seccomp filter? Andy wrote it.
-    <li class="nagra">
-      <h3>Jasvir Nagra</h3>
-      <h4>Web Security Expert</h4>
-      <p>Jas is a web security expert, formerly of the Google security team, where among other things he led the <a href="https://developers.google.com/caja/">Caja</a> project to create a variant of Javascript that enforces capability-based security.
-    <li class="seaborn">
-      <h3>Mark Seaborn</h3>
-      <h4>Linux Sandboxing Expert</h4>
-      <p>Mark is a Linux kernel security and sandboxing expert on the Google Chrome team, where among other things he helped build the NaCl sandbox and <a href="http://googleprojectzero.blogspot.com/2015/03/exploiting-dram-rowhammer-bug-to-gain.html">developed the first working exploit of the Rowhammer hardware security flaw</a>. Previous to Google, Mark developed <a href="http://plash.beasts.org/contents.html">Plash</a>, a capability-based sandbox for Unix shell commands.
-    <li class="miller">
-      <h3>Mark S. Miller</h3>
-      <h4>Distributed Security Expert</h4>
-      <p>Mark is one of the leading experts -- arguably <i>the</i> leading expert -- in the field of capability-based security. He is the designer of the <a href="http://erights.org/">E Language</a> and the <a href="http://erights.org/elib/distrib/captp/index.html">CapTP protocol</a> on which <a href="https://capnproto.org">Cap'n Proto</a> (Sandstorm's underlying communications layer) is based.
-  </ul>
-</section>
+<p>In early 2017, Sandstorm Development Group <a href="https://sandstorm.io/news/2017-02-06-sandstorm-returning-to-community-roots">ran out of funding</a> and the team primarily <a href="https://sandstorm.io/news/2017-03-13-joining-cloudflare">joined Cloudflare</a>. Sandstorm development continued with a major rewrite of the HTTP proxy, a refactor of the account model, and the addition of localization support. However, the pace of development slowed, and the Oasis <a href="https://sandstorm.io/news/2018-08-27-discontinuing-free-plan">free plan was discontinued</a> in 2018. Finally, <a href="https://sandstorm.io/news/2019-09-15-shutting-down-oasis">Oasis shut down</a> for paying customers at the end of 2019.</p>

```

Comment:
Reading the article start to finish, it's a little confusing that the team ostensibly disbanded but development continued. I think at this point it was a "community project" right? Even before 2020. Maybe clarify that?